Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -1105,6 +1194,7 @@ namespace gdjs {
       _authenticationLoaderContainer = null;
       _authenticationIframeContainer = null;
       _authenticationTextContainer = null;
+      removeAutomaticGamesPlatformAuthenticationCallback();
```

Comment:
Not entirely sure why we need this here?

Reply:
I've improved the logic and removed that part